pkg/controllers/externalsecret/externalsecret_controller.go

```diff
@@ -332,7 +330,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	}, nil
 }
 
-func patchSecret(ctx context.Context, c client.Client, scheme *runtime.Scheme, secret *v1.Secret, mutationFunc func() error) error {
+func patchSecret(ctx context.Context, c client.Client, scheme *runtime.Scheme, secret *v1.Secret, mutationFunc func() error, fieldOwner string) error {
```

Comment:
Can we make this a fqdn? Like `foo.external-secrets.io` that way it becomes clearer who ownes this particular field.

Reply:
Huh, this is a good question, I'll need to look it up 😆 I just rushed to fix the issue actually. I'll give a search at kubectl github and try to follow the same standard.
If I don't find any, I'll just FQDN it away. It is better anyways for traceability.

Reply:
It seems that they allow a configuration from the CLI to any value, and defaults to these ones: https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/apply/apply.go#L757.

I'll FQDN it :)